widget_driver/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.0.3
+
+* Removes `FlowCoordinators` since it is currently not used. We will add this back if it ever becomes needed.
```

Comment:
I added these `FlowCoordinators` in the beginning of WidgetDriver.
The idea was that we can handle navigation in them. So that views/widgets do not need to do the push and pop and know about other widgets.

But I never added anymore code to these.

In the future they might come back. But currently there is no need for them. So I just cleaned up the code and removed them